src/tests/fuzzing/fuzz_pkcs15_decode.c

```diff
@@ -20,60 +20,92 @@
 #include "config.h"
 #endif
 
+#include "fuzzer_reader.h"
 #include "libopensc/pkcs15.h"
 #include "libopensc/internal.h"
-#include <stdlib.h>
-#include <string.h>
 
-static struct sc_context *ctx = NULL;
-static struct sc_pkcs15_card *p15card = NULL;
-static sc_card_t card = {0};
+const char *__asan_default_options() {
+  return "verbosity=0:mallocator_may_return_null=1";
+}
```

Comment:
This part deserves a comment why it is needed now.

Reply:
Yes, I believe it should be in the fuzzers that use some memory allocation (with this flag, the allocator will return 0 on out-of-memory, according to [SanitizerCommonFlags](https://github.com/google/sanitizers/wiki/SanitizerCommonFlags)) - I already added it into the remaining two with some describing comments.

Reply:
And do we need the `verbosity` one? It should be default. The other should be called `allocator_may_return_null` (without the `m`), but I am afraid this will cause lot of failures during the allocations before we will get to a useful code, even though it might increase code coverate with the error code paths.

I would probably keep this out, at least from this PR and at least for now.

Reply:
Ok, understood, I removed it.